inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -91,6 +102,9 @@ def simulate(
 
         task.set_output_class(ProteinSolvationOutput)
 
+        if compressed_trajectory is True:
```

Comment:
Remove this if and just keep the assignement you are doing below.

The idea is now that the task will handle everything! Then if you do:
`task.get_output(all_files=True)` it will download all files if with False, it will just download the default file.

Now the main idea is that you need to manage appropriately in the ProteinSolvationOutput. In the WindTunnelOutput (more precisely, on the SteadyStateOutput) we are inspecting if the simulation files we download match or not with the default files (in order to avoid passing any further arguments). Then, the functions of the class execute the appropriate steps to use the correct files. 

Notice that, you may need other files for the rendering and you need to be aware of the other functions as well. At least trow some errors for users to understand things are not available.